testes/potigol/avaliador-sintatico.test.ts

```diff
@@ -35,7 +35,7 @@ describe('Avaliador sintático', () => {
                     expect(retornoAvaliadorSintatico.declaracoes).toHaveLength(1);
                 });
     
-                it('Sucesso - Operações encadeadas', () => {
+                it.only('Sucesso - Operações encadeadas', () => {
```

Comment:
@samuelrvg  esse `it.only` deveria continuar aqui?

Reply:
@pablotdv Na branch principal esta normal, sem o `.only` pelo que ví aqui.

https://github.com/DesignLiquido/delegua/blob/449fb55ddeb245fb56c609184ee54e74226ef97d/testes/potigol/avaliador-sintatico.test.ts#L38C61-L38C61